autoload/gh/github/repos.vim

```diff
@@ -51,3 +51,12 @@ function! gh#github#repos#create(data) abort
         \ }
   return gh#http#request(settings)
 endfunction
+
+function! gh#github#repos#get_repo(owner, repo) abort
+  let url = printf('https://api.github.com/repos/%s/%s', a:owner, a:repo)
+  let settings = {
+        \ 'method': 'GET',
+        \ 'url': url,
+        \ }
+  return gh#http#request(settings)
```

Comment:
If just request GET, you can use `gh#http#get()` to get it.

Reply:
thanks, fixed by [this](https://github.com/skanehira/gh.vim/pull/87/commits/572b042025678c5e9f15864dcce366db71a4dedf) commit.